actions/authenticated.js

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
why this action is needed?

Reply:
just as an example how to use authenticated middleware and for unit tests.